ui/app/src/main/java/com/example/fuel/model/User.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.fuel.model
+
+data class User (

```

Comment:
It shouldn't be named User, because this is model that will be used only for registration so better name would be RegistrationModel or something like this.